README.md

```diff
@@ -112,27 +111,27 @@ If you would like to contribute to the list of Platform.sh's maintained template
 
 Let's use the previous example: you have created a new application that uses the framework Spiffy that you think would be a useful template.
 
-1. Create a repository under the [Platform.sh Github organization](https://github.com/platformsh/) for the template. Officially maintained templates begin with the `template-` prefix, so name it `template-spiffy`, and initialize it with an empty `README.md`.
-2. Add the "Examples and Templates team" as `Admin` collaborators in the `platformsh/template-spiffy` repository settings.
-3. Integrations: The DevRel team will create an integration during the review process, so you don't need to worry about having one set up, so long as you have given the "Examples and Templates team" "Admin" access to the repository.
-4. Clone the [`template-builder` repository](https://github.com/platformsh/template-builder) locally. Create and checkout a new branch called `add-spiffy`.
-5. Each template project is in its own directory within `templates`, which corresponds to a GitHub repository with the `template-` prefix. Create the directories `templates/spiffy` and `templates/spiffy/files/` on the `add-spiffy` branch.
-6. Add only the files for `template-spiffy` into `templates/spiffy/files`, as dependency downloads and linking to an upstream repository can be handled by the Python build process. For example, the Drupal 8 template links to its upstream [here](https://github.com/platformsh/template-builder/blob/new-template-instructions/project/drupal.py). If your application requires any patches to deploy on Platform.sh, copy them into `templates/spiffy/`.
-7. Include or update the `README.md` so that it is similar to other templates. Address any information specific to running the application on Platform.sh you think the customer should know.
-8. Each template comes with a file called `.platform.template.yaml`, which is used to define how the template repository will appear in and initialize from the management console. See the example in the [external templates](https://github.com/platformsh/templates-external/blob/master/template-definition.yaml) repo for instructions.
+1. Create a repository under the [Platform.sh Templates organization](https://github.com/platformsh-templates/) for the template.  Create a repository named `spiffy` and initialize it with an empty `README.md`.  If you don't have access, ping someone on the DevRel team to create it for you.
```

Comment:
Ugh.  Good point.  The problem is that template-builder needs the repo to have something in it already; it can't be an empty repo.  Can you set the default branch when creating the repo?

Reply:
No, you can't change the default branch when creating the repo.
What I did is create an empty repo so no default branch is set, then I manually pushed the README to it on a master branch which became the default automatically since it's the only branch.